docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -71,6 +71,18 @@ the applications you want.
   >
   > This feature supports Tizen 5.0 and Higher for Mobile and Wearable.
 
+- Updating and retrieving abilities
+
+  You can [update the abilities](#updating-and-retrieving-abilities) on the server side, and then retrieve the abilities on the client side.
+
+  The media controller server provides the update and retrieval abilities to the registered application.
```

Comment:
@mijin85cho thanks,  
1. I do agree with you.  
2. Why I added  "update and retrieval" is because, the whole section is talking about two abilities, which have multiple associated callbacks. 

@haejeongkim as requested by @mijin85cho , can you please provide your valuable opinion on this.

Reply:
The media controller server provides current abilities about the registered application.
=> I updated like this.